src/api/index.test.js

```diff
@@ -1,56 +1,234 @@
+        organisation_name: "Nelson's Horsenettle",
+        phone: '+49 543 513 8358',
+        website: 'www.nelson.de',
+        description: 'Everyday, 24/7',
+      },
+      {
+        _id: '5f9db611c7cc881787ba620a',
+        city: 'Berlin',
+        country: 'Germany',
+        organisation_name: 'Test name',
+        phone: '+49 543 510 8358',
+        website: 'www.test.de',
+        description: 'Everyday, 24/7',
+      },
+      {
+        _id: '5f9db611c7cc881787ba620b',
+        city: 'Berlin',
+        country: 'Germany',
+        organisation_name: 'Test name 2',
+        phone: '+49 543 514 8358',
+        website: 'www.test2.de',
+        description: 'Everyday, 24/7',
+      },
+    ];
+    // when
+    mockFMClient
+      .onGet('/hotlines', { params: { searchTerm: search } })
+      .reply(200, response);
+    const hotlines = await appApiClient.getHotlinesData(search);
+    // then
+    expect(hotlines.data).toEqual(response);
+  });
+
+  it('should successfully fetch shelters data', async () => {
+    // given
+    const response = [
+      {
+        place_name: 'Test name',
+        address: '10001, Berlin, TestStrasse, 1',
+        contact_person: 'Jon Snow',
+        phone: '+4900110000',
+        locs: [12.2343, 13.7898],
+      },
+    ];
+    // when
+    mockFMClient.onGet('/shelters').reply(200, {
+      data: response,
+    });
+    const shelters = await appApiClient.getSheltersData();
+    // then
+    expect(shelters.data.data).toEqual(response);
+  });
+
+  it('should successfully fetch articles data', async () => {
+    // given
+    const response = [
+      {
+        title: 'Test title',
+        author: 'Test User',
+        text: 'Lorem ipsum',
+        violence_type: ['emotional'],
+        url_to_image: '"https://www.google.com/',
+        // created_at: new Date(), TODO: fix handling dates
+      },
+    ];
+    // when
+    mockFMClient.onGet('/articles').reply(200, {
+      data: response,
+    });
+    const articles = await appApiClient.getArticlesData();
+    // then
+    expect(articles.data.data).toEqual(response);
+  });
+
+  it('should successfully fetch articles data by id', async () => {
+    // given
+    const response = [
+      {
+        title: 'Test title',
+        author: 'Test User',
+        text: 'Lorem ipsum',
+        violence_type: ['emotional'],
+        url_to_image: '"https://www.google.com/',
+        // created_at: new Date(), TODO: fix handling dates
+      },
+    ];
+    const id = '6062e6501e80a94test40522';
+    // when
+    mockFMClient.onGet(`/articles/${id}`).reply(200, {
+      data: response,
+    });
+    const article = await appApiClient.getArticleById(id);
+    // then
+    expect(article.data.data).toEqual(response);
+  });
+
+  it('should successfully send user data on POST to /login endpoint', async () => {
+    const email = 'test@test.com';
+    const password = '12345678';
+    // given
+    const user = {
+      username: 'Celeste',
+      email,
```

Comment:
should we have password here ?

Reply:
i think no, also do we have password in response? why? we need token only

Reply:
i think we have password in the response since we send the whole user document + the token

Reply:
i think we should only send token no user

Reply:
not password for sure, because anyway fe uses only token for auth